docs/source/conf.py

```diff
@@ -307,7 +307,7 @@ def setup(app):
 # https://stackoverflow.com/questions/15889621/sphinx-how-to-exclude-imports-in-automodule
 
 MOCK_REQUIRE_PACKAGES = []
-with open(os.path.join(PATH_ROOT, 'requirements.txt'), 'r') as fp:
+with open(os.path.join(PATH_ROOT, 'requirements-extra.txt'), 'r') as fp:
```

Comment:
can split running the test and making docs, I mean that for just build doc via ReadTheDocs we would use just mock and for other CI like CircleCI we would do the full import...
I guess that there as to be a flag which distinguishes between `make html` and `make tests`

Reply:
there is a RTD env variable called `READTHEDOCS` we could check
https://docs.readthedocs.io/en/stable/builds.html

Reply:
ok, on the other hand, the RTD docs if build only with push to master which is not so often
so let's keep it as it is now and if we find that the build-time is an issue we'll change it

Reply:
ok good, in case we need it I can do it quickly